Module/STIG/Functions.Checklist.ps1

```diff
@@ -258,47 +279,47 @@ function Get-VulnerabilityList
+                [PSCustomObject]@{ Name = 'IA_Controls'; Value = $vulnerabiltyDiscussionElement.discussionroot.IAControls },
+                [PSCustomObject]@{ Name = 'Check_Content'; Value = $vulnerability.Rule.check.'check-content' },
+                [PSCustomObject]@{ Name = 'Fix_Text'; Value = $vulnerability.Rule.fixtext.InnerText },
+                [PSCustomObject]@{ Name = 'False_Positives'; Value = $vulnerabiltyDiscussionElement.discussionroot.FalsePositives },
+                [PSCustomObject]@{ Name = 'False_Negatives'; Value = $vulnerabiltyDiscussionElement.discussionroot.FalseNegatives },
+                [PSCustomObject]@{ Name = 'Documentable'; Value = $vulnerabiltyDiscussionElement.discussionroot.Documentable },
+                [PSCustomObject]@{ Name = 'Mitigations'; Value = $vulnerabiltyDiscussionElement.discussionroot.Mitigations },
+                [PSCustomObject]@{ Name = 'Potential_Impact'; Value = $vulnerabiltyDiscussionElement.discussionroot.PotentialImpacts },
+                [PSCustomObject]@{ Name = 'Third_Party_Tools'; Value = $vulnerabiltyDiscussionElement.discussionroot.ThirdPartyTools },
+                [PSCustomObject]@{ Name = 'Mitigation_Control'; Value = $vulnerabiltyDiscussionElement.discussionroot.MitigationControl },
+                [PSCustomObject]@{ Name = 'Responsibility'; Value = $vulnerabiltyDiscussionElement.discussionroot.Responsibility },
+                [PSCustomObject]@{ Name = 'Security_Override_Guidance'; Value = $vulnerabiltyDiscussionElement.discussionroot.SeverityOverrideGuidance },
                 [PSCustomObject]@{ Name = 'Check_Content_Ref'; Value = $vulnerability.Rule.check.'check-content-ref'.href },
-                [PSCustomObject]@{ Name = 'Weight'; Value = $vulnerability.Rule.Weight},
-                [PSCustomObject]@{ Name = 'Class'; Value = 'Unclass'},
-                [PSCustomObject]@{ Name = 'STIGRef'; Value = "$($XccdfBenchmark.title) :: $($XccdfBenchmark.'plain-text'.InnerText)"},
-                [PSCustomObject]@{ Name = 'TargetKey'; Value = $vulnerability.Rule.reference.identifier}
+                [PSCustomObject]@{ Name = 'Weight'; Value = $vulnerability.Rule.Weight },
+                [PSCustomObject]@{ Name = 'Class'; Value = 'Unclass' },
+                [PSCustomObject]@{ Name = 'STIGRef'; Value = "$($XccdfBenchmark.title) :: $($XccdfBenchmark.'plain-text'.InnerText)" },
+                [PSCustomObject]@{ Name = 'TargetKey'; Value = $vulnerability.Rule.reference.identifier }
 
                 # Some Stigs have multiple Control Correlation Identifiers (CCI)
                 $(
                     # Extract only the cci entries
                     $CCIREFList = $vulnerability.Rule.ident |
-                        Where-Object {$PSItem.system -eq 'http://iase.disa.mil/cci'} |
-                            Select-Object 'InnerText' -ExpandProperty 'InnerText'
+                    Where-Object { $PSItem.system -eq 'http://iase.disa.mil/cci' } |
```

Comment:
Whitespace inside of curly brackets.
Also could you put back the indention in front of select-object.  We use the indention to indicate the line continuation.